frontend/src/i18n/locales/en.yaml

```diff
@@ -283,6 +283,7 @@ manage:
     table:
       group: Group
       user: User
+      yourself: Yourself
```

Comment:
Is "yourself" really the proper word to use here? Is it more appropriate than the shorter "you"? I really don't know. I might ask around, just commenting here so that I won't forget.

Reply:
David suggested "Yourself" but I also don't mind "You"

Reply:
I asked a native friend. Will report back once I have an answer.

Reply:
She said "You", i.e. what you have pushed now -> resolving.